SpriterDotNet/Parser/XmlSpriterParser.cs

```diff
@@ -24,7 +26,14 @@ public Spriter Parse(string data)
 
         public bool CanParse(string data)
         {
-            return data.StartsWith(XmlStart);
+            return data != null && data.StartsWith(XmlStart);
```

Comment:
My issue with that is that by doing that you are locking the consumer to always use the static SpriterParser if they want null checking etc. 

Including it in each XyzSpriterParser implementation is slightly less DRY, but it reduces assumptions and documents through code that null is an invalid value in a function that is perfectly named for the check.

If you want to stick with the null checks in SpriterParser, that's fine, I just want you to understand where I'm coming from with this chain of thought.